Authentication/SignUp/View/SignupControllerDelegate.swift

```diff
@@ -41,7 +41,7 @@ extension SignupControllerDelegate {
     public var shouldDisplaySignupErrorWithAlert: Bool { return true }
     
     public func signupControllerWillExecuteSignUp(controller: SignupController) {
-        if let errorLabel = controller.signupView.signupErrorLabel {
+        if let errorLabel = controller.signupView.signUpErrorLabel {
             errorLabel.text = " "
```

Comment:
For it to clean previous errors of signup, so that they can be distinguished from the possible new ones.